src/test/groovy/org/gradle/profiler/mutations/CopyFileMutatorTest.groovy

```diff
@@ -0,0 +1,29 @@
+package org.gradle.profiler.mutations
+
+class CopyFileMutatorTest extends AbstractMutatorTest {
+
+    def "copies file from source to target"() {
+        def testDir = tmpDir.newFolder()
+
+        def expectedContents = "Copy file from source to target"
+        def source = new File(testDir, "source.txt")
+        source.text = expectedContents
+        def target = new File(testDir, "nested/target.txt")
+
+        def spec = mockConfigSpec("""{
+            copy-file {
```

Comment:
❔ Can we copy also multiple files or we can copy just one?

Reply:
We can copy one source (a file or a directory) to a target location.